extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -166,9 +166,20 @@ export class AmpAnalytics extends AMP.BaseElement {
       this.analyticsGroup_.dispose();
       this.analyticsGroup_ = null;
     }
+  }
+
+  /** @override */
+  resumeCallback() {
+    this.initIframeTransport_();
+  }
+
+  /** @override */
+  unlayoutCallback() {
     if (this.iframeTransport_) {
```

Comment:
@dvoytenko Can you confirm this, please?

Reply:
I believe unlayout is fine. In most of cases, when embed is removed, detachedCAllback is not called either.

Reply:
@dvoytenko In my testing, I am seeing that unlayoutCallback() is getting called immediately, and resumeCallback() is not getting called. Therefore, in sendRequest() (approx line 800 below) we are getting a null exception on this.iframeTransport_.sendRequest(request);
Is this because amp-analytics is not visible on the page?
If so, I propose removing unlayoutCallback() and resumeCallback().

Reply:
No, `resumeCallback` should be called always. Please ping @jridgewell to check on this.

Reply:
Ok, thanks. I'm about to push a change that:
1. Adds resumeCallback() and unlayoutCallback() back, exactly as they were 24 hours ago in this PR
2. Removes setting iframeTransport_ to null in detachedCallback(), since that was only added when I removed unlayoutCallback() yesterday
3. Upgrades the null check on iframeTransport_ in sendRequest_() to an assert() instead.

To reproduce the issue I'm seeing:
1. Run gulp
2. Comment out the XSS safety check in examples/analytics-iframe-transport-remote-frame.html, so that the stuff inside the "if" always runs.
3. Hit http://localhost:8000/examples/analytics-iframe-transport.amp.html directly